src/rmrk2.0.0/tools/consolidator/utils.ts

```diff
@@ -7,25 +7,35 @@ import {
 import { Collection } from "../../classes/collection";
 import { IConsolidatorAdapter } from "./adapters/types";
 import { Base } from "../../classes/base";
+import { List } from "../../classes/list";
 import { changeIssuerInteraction } from "./interactions/changeIssuer";
 import { ChangeIssuer } from "../../classes/changeissuer";
 import { Remark } from "./remark";
 import { OP_TYPES } from "../constants";
 
-export const isNftTransferable = (nft: NFT, remark: Remark) => {
-  return (
-    nft.transferable === 1 ||
-    (nft.transferable < 0 && nft.block - nft.transferable >= remark.block) ||
-    (nft.transferable > 1 && remark.block >= nft.transferable)
-  );
+export const isNftTransferable = (nft: NFT, remark: Remark, opType: OP_TYPES) => {
```

Comment:
* please add some docs comments about extra logic and unit tests.
* prefer strict equality / non equality
* multiple `else` conditions could be omitted

Reply:
Added comments and changed to strict equal but by removing else conditions i'd change the logic and remove the changes i made

Reply:
You were right i removed one of the else conditions